ignite/cmd/scaffold_map.go

```diff
@@ -13,10 +13,49 @@ const (
 // NewScaffoldMap returns a new command to scaffold a map.
 func NewScaffoldMap() *cobra.Command {
 	c := &cobra.Command{
-		Use:   "map NAME [field]...",
-		Short: "CRUD for data stored as key-value pairs",
-		Args:  cobra.MinimumNArgs(1),
-		RunE:  scaffoldMapHandler,
+		Use: "map NAME [field]...",
+		Short: `The "map" scaffolding command is used to generate files that implement the logic
```

Comment:
@fadeev oh I just noticed that you should use the `Long` field for such description, because that makes the `ignite scaffold` command really chatty!